components/brave_rewards/browser/rewards_service_impl.h

```diff
@@ -374,6 +372,8 @@ class RewardsServiceImpl : public RewardsService,
 
   void EnableGreaseLion();
 
+  std::string GetExternalWalletType() const;
```

Comment:
not needed

Reply:
@tmancey it was already added as private function.

Reply:
i just reverted the change to keep as it was before.